lib/boom_notifier/application.ex

```diff
@@ -6,7 +6,11 @@ defmodule BoomNotifier.Application do
   def start(_type, _args) do
     import Supervisor.Spec
 
-    children = [supervisor(BoomNotifier.ErrorStorage, [])]
+    children = [
+      supervisor(BoomNotifier.ErrorStorage, []),
+      {BoomNotifier.NotifierSenderServer, nil}
+    ]
+
     Supervisor.start_link(children, strategy: :one_for_all)
```

Comment:
Is there any need of restarting the `NotifierSenderServer` when the `ErrorStorage` crashes or the other way around? I might be missing something but so far they seem independent to me :thinking: 

If so, would it be more appropriate to go with [`strategy: :one_for_one`](https://hexdocs.pm/elixir/1.12/Supervisor.html#module-strategies)?